js/ui/codeView.js

```diff
@@ -514,6 +509,16 @@ var CodingSession = GObject.registerClass({
         return this._toolbox;
     }
 
+    get toolboxId() {
+        // FIXME: this should be extended to make it possible to launch
+        // arbitrary toolboxes in the future, depending on the application
+        if (this.appId.startsWith('com.endlessm.')) {
+            return 'com.endlessm.HackToolbox';
+        }
```

Comment:
Style: single line `if`s don't need curly braces

Also, allow me to suggest the following:

```javascript
let prefix = 'com.hack_computer';

if (this.appId.startsWith('com.endlessm.'))
    prefix = 'com.endlessm';

return `${prefix}.HackToolbox`;
```

Reply:
Okay, I've just follow your suggestion here and use the prefix code.